Chirp/Login Page/LoginPageViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  LoginPageViewController.swift
+//  Chirp
+//
+//  Created by ioannis on 7/6/23.
+//
+
+import UIKit
+import PhoneNumberKit
+
+class LoginPageViewController: UIViewController {
+  
+    private let titleLabel:UILabel = {
+        let label = UILabel()
+        label.text = "What’s your phone number?"
+        label.numberOfLines = 0
+        label.font = .systemFont(ofSize: 32,weight:.bold)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .label
+        return label
+    }()
+    
+    private let subTitleLabel:UILabel = {
+        let label = UILabel()
+        label.text = "We’ll send you a code to verify your number"
+        label.numberOfLines = 0
+        label.font = .systemFont(ofSize: 14,weight:.light)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .label
+        return label
+    }()
+    private let numberTextField : PhoneNumberTextField = {
+       let textField = PhoneNumberTextField()
+        let phoneNumberKit = PhoneNumberKit()
+        textField.borderStyle = .roundedRect
+        textField.keyboardType = .namePhonePad
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        textField.placeholder = "Enter your mobile number"
```

Comment:
Instead of showing the constant placeholder can we use a property of this library to show an example instead?
Also set the text type to phone number field so that it shows a number pad instead of characters

```suggestion
        textField.withExamplePlaceholder = true
        textField.placeholder.withDefaultPickerUI = true
        textField.textContentType = .telephoneNumber
```